flink-end-to-end-tests/test-scripts/test_kubernetes_session.sh

```diff
@@ -29,6 +29,7 @@ ARGS="--output ${OUTPUT_PATH}"
 function cleanup {
     kubectl delete service ${CLUSTER_ID}
     kubectl delete clusterrolebinding ${CLUSTER_ROLE_BINDING}
+    stop_kubernetes
```

Comment:
This means that for every K8s test we start and stop `minikube`. This might add some time.

Reply:
Nice suggestion. I will create a follow-up ticket to optimize the start/stop of minikube.

BTW, i think the Yarn/Mesos/Kubernetes share the logics. Start a new cluster for each e2e test.